scripts/functional-tests.sh

```diff
@@ -324,7 +324,7 @@ test_model_print(){
         set timeout 30
         spawn ilab chat -m bar
         expect {
-            "Welcome to Chat CLI w/ BAR" { exit 0 }
+            "Welcome to InstructLab Chat w/ BAR" { exit 0 }
```

Comment:
"Welcome to InstructLab Chat CLI w/" still need the CLI

Reply:
Ah, thanks @bjhargrave !

Reply:
fixed for real this time, i hope ...

Reply:
```
╭─────────────────────────────────── system ───────────────────────────────────╮
  │ Welcome to InstructLab Chat w/ BAR (type /h for help)                        │
  ╰──────────────────────────────────────────────────────────────────────────────╯
```

This is what I'm seeing in the log, and the code ... where did you get "InstructLab Chat CLI" ?